notebooks/README.md

```diff
@@ -1,22 +1,8 @@
-
```

Comment:
I don't think just this readme is enough to recreate the task that you performed. However, the instructions in the colab notebook are great! Like I suggest later, maybe add the notebook and refer to that for replication instructions.

Reply:
yeay. I did it. thanks.